Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/cards/IncidenceCard.kt

```diff
@@ -23,6 +27,23 @@ class IncidenceCard(parent: ViewGroup) :
         item: StatisticsCardItem,
         payloads: List<Any>
     ) -> Unit = { item, payloads ->
-        item.stats as IncidenceStats
+
+        viewBinding.value.iconInfo.setOnClickListener {
+            item.onHelpAction.invoke(item.stats)
+        }
+
+        val stats = item.stats as IncidenceStats
+
+        val primaryFigure =
+            stats.keyFigures.firstOrNull { it.rank == KeyFigureCardOuterClass.KeyFigure.Rank.PRIMARY }
```

Comment:
`IncidenceStats` has now getter `sevenDayIncidence` and it's always not null

Reply:
That's nice. I updated the PR.